### DISCUSSION TITLE
Edit problem with complex data hierarchy in w2grid

### DISCUSSION THREAD

**Discussion by riteshkapse** [vitmalina/w2ui#1983] [Q&A]
I've column definition like:
```
{ caption: 'ID#', field: 'myForm.checkDigitId',frozen: true, min: 200 } // Note the DOT in field
```

Data record like:
```
rec : {
     myForm: {
           checkDigitId : "1111"
     }
}
```

In above case when i click on grid cell to edit, the past value is lost and I always get empty text field.

It looks very basic requirement to me. Am i missing something in config?

I am trying to fix this in my fork.

**Comment by vitmalina** [comment]
It should be working. Please create jsFiddle to demonstrate the problem.

**Reply by riteshkapse** [reply]
Hi Vitmalina :)

Thanks for response.

Check this fiddle http://jsfiddle.net/6t527sje/ . Try editing "text" column cells.

I could fix it in w2grid.js as below. Please advise if its fine.
![image](https://user-images.githubusercontent.com/37905230/112259827-ee86fa00-8c8e-11eb-808a-efc218de106e.png)
